guides/common/assembly_registering-hosts.adoc

```diff
@@ -11,6 +11,10 @@ include::modules/con_registering-hosts-by-using-global-registration.adoc[levelof
 
 include::modules/ref_global-parameters-for-registration.adoc[leveloffset=+2]
 
+ifdef::katello,orcharhino,satellite[]
+include::modules/proc_configuring-a-host-for-registration.adoc[leveloffset=+2]
+endif::[]
```

Comment:
I'm including this procedure for non-Katello products only because time synchronization wasn't required and I was told that vanilla Foreman doesn't care about the certificate.

Reply:
> for non-Katello products

Isn't it the opposite, `ifdef::katello,orcharhino,satellite[]` is for katello, right?

Reply:
@stejskalleos That's correct.